internal/pkg/service/common/etcdop/serde/serde.go

```diff
@@ -0,0 +1,66 @@
+// Package serde provides encode, decode and validate operations for any value.
+package serde
```

Comment:
Pred implementaciou iteratora som potreboval presunut serializaciu do samostatneho balicka, aby sa mi lepsie pisali testy. 

Zvolil som nazov `serde`, kedze `serialization.Serialization` mi prislo dlhe.

Tato skratka sa myslim bezne pouziva https://serde.rs/

Reply:
To je hroznej svět, tohle. 😅

Reply:
Ta skratka? :smile:

Reply:
No myslím tady celej ten gočkovej/rustovej ekosystém zkratek. 😆

Reply:
Ale tahle je jedna z těch nejpovedenějších. 🤣

Reply:
Kamarat robi pre nemecku banku a ukazoval minule nazov balicka v Jave zlozeny z 10 nemeckych slov :slightly_smiling_face: , toto mi pride mensi extrem.